.github/workflows/main.yml

```diff
@@ -101,11 +100,9 @@ jobs:
           CURRENT_DATE=$(date +'%Y-%m-%dT%H_%M_%S') # no colons allowed for artifacts
           echo "date=$CURRENT_DATE" >> $GITHUB_OUTPUT
 
-      - uses: actions/upload-artifact@v3
-        if: always()
-        with:
-          name: unit_results_${{ matrix.python-version }}-${{ steps.date.outputs.date }}.csv
-          path: unit_results.csv
```

Comment:
Do we want to delete this step?  It's actually the output from the tests.  It's not something I've used previously but maybe someone is?

Reply:
If we don't need it, I'm happy with yanking it.